File: CS/RibbonDesigner/XtraReport1.cs
using System;
using DevExpress.XtraPrinting;
using DevExpress.XtraReports.UI;
// ...

namespace RibbonDesigner {
    public partial class XtraReport1 : XtraReport {
        public XtraReport1() {
            InitializeComponent();
            PrintingSystem.SetCommandVisibility(PrintingSystemCommand.Save, CommandVisibility.None);
        }
    }
}

File: VB/RibbonDesigner/XtraReport1.vb
Imports Microsoft.VisualBasic
Imports System
Imports DevExpress.XtraPrinting
Imports DevExpress.XtraReports.UI
' ...

Namespace RibbonDesigner
	Partial Public Class XtraReport1
		Inherits XtraReport
		Public Sub New()
			InitializeComponent()
			PrintTool.MakeCommandResponsive(Me.PrintingSystem)
			PrintingSystem.SetCommandVisibility(PrintingSystemCommand.Save, CommandVisibility.None)
		End Sub
	End Class
End Namespace
